docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
   - Mandatory
-* - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
-  - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
-  - Recommended
-* - You must record logs for all TRE components and actions, including systems, relevant applications, networks, user access and data access.
-  - Logging is an critical part of ensuring all IT infrastructure is secure and operating as expected, as well as for investigating problems.
+* - You must use reported issues to inform changes, such as for process improvement and risk management.
+  - All issues should be analysed for their route cause and improvements put in place to prevent further occurrence.
   - Mandatory
-* - You must store logs in a secure location only accessible to authorised administrators.
-  - Logs often contain sensitive information.
+```
+
+**Quality Management Data:** Data, including training records and configuration data, collected and used to monitor, evaluate, and improve the quality of processes, or services within the TRE organisation.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-quality-management-data
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must collect and maintain quality management data for measuring the effectiveness of a TRE.
+  - Large amounts of data will be produced by elements within the TRE.
+    These data should be analysed with reports and dashboards provided to guide TRE implementer’s improvements and provide re-assurance to data consumers and subjects.
   - Mandatory
 ```
 
-## Risk management
+**Electronic Quality Management System Application:** A software application or platform used to manage and automate quality management processes, including document control, corrective actions, audits, and performance tracking.
 
-_The ability of the TRE organisation to measure, forecast and evaluate risks to information._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-electronic-quality-management-system-application
+
+* - Statement
+  - Guidance
+  - Importance
+* - You could use an eQMS (Electronic Quality Management System) to standardise and automate quality management tasks and workflows, and to generate quality data and reports automatically.
+  - A basis eQMS could be a set of spreadsheets or documents held in a repository which are manually maintained.
+    More mature applications will provide workflows and generate quality data through manual and automated actions.
+  - Optional
+```
+
+## Risk Management
+
+What the organisation does to ensure information risk is measured and managed to an acceptable level.
+
+**Risk Assessment:** The systematic evaluation and analysis of potential risks, threats, or vulnerabilities, including their likelihood, potential impact, and the effectiveness of existing controls or mitigation measures.
```

Comment:
Modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#risk-management , split into separate assessment and treatment sections